examples/chat/client/client.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
```

Comment:
Lets add the license header.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-websocket/pull/276/commits/c58194313ce39c7e9712c55b6fb5b19f99570993